packages/round-manager/src/features/round/ViewApplicationPage.tsx

```diff
@@ -173,12 +195,21 @@ export default function ViewApplicationPage() {
       setOpenProgressModal(true);
       setOpenModal(false);
 
-      application.status = reviewDecision;
+      if (reviewDecision == "IN_REVIEW") {
+        application.inReview = true;
+      } else {
+        application.status = reviewDecision;
+      }
 
       await bulkUpdateGrantApplications({
         roundId: roundId,
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         applications: applications!,
+        payoutAddress:
+          reviewDecision == "IN_REVIEW"
+            ? // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+              applications![0].payoutStrategy?.id
```

Comment:
If applications is never null, that should be reflected by the type of `useApplicationByRoundId`. If it can be null, it should be checked rather than being asserted as non-null.

If it can be an empty array at any point, the above will throw an error at runtime.